gulp/config.js

```diff
@@ -1,11 +1,65 @@
-'use strict';
+var dest = "./dist/assets/";
+var src = "./src/assets";
+var version = "/1.0.0";
 
 module.exports = {
+  concatJS: {
+    documentation: {
+      src: [
+        src + '/js/site/global.js',
+        src + '/js/site/prism.js'
+      ],
+      dest: dest + '/js'
+    },
+    protocol: {
+      src: [
+        src + '/js/protocol/*.js'
+      ],
+      dest: dest + '/js'
+    }
+  },
+  lintCss: {
+    src: [
+      dest + "/sass/protocol/protocol.css",
+      dest + "/sass/site/site.css"
+    ]
+  },
+  cleanCss: {
+    src: [
+      dest + '/css/protocol.css',
+      dest + '/css/site.css',
+      dest + '/css/prism.css'
+    ],
+    dest: dest + '/css',
+    rename: { suffix: '.min' }
+  },
+  replace: {
+    src: src + '/sass/protocol/protocol.scss',
+    dest: dest + '/css',
+    base: "../../../../",
+    replacement: "../../../bower_components/"
+  },
+  sass: {
+    src: [
+      src + '/sass/protocol/protocol.scss',
+      src + '/sass/site/site.scss',
+      src + '/sass/site/prism.scss'
```

Comment:
Still missing out on demo styles here. And I still think it makes sense to process the whole folder and not individual files. I think it just means switching this to `src + '/sass/**/*.scss'` (and updating a few similar paths in other places).

Reply:
Switching this to `src + '/sass/**/*.scss` is causing the build to generate duplicated scss files. So I included the demos scss file and I'll create a task to come back to this issue.
![image](https://user-images.githubusercontent.com/5924575/38703284-b7b06300-3e57-11e8-977c-bf23b7447d65.png)